src/main/java/net/adoptium/GreetingResource.java

```diff
@@ -1,16 +1,35 @@
 package net.adoptium;
 
 import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
+import java.util.Locale;
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
 
 @Path("/hello")
 public class GreetingResource {
 
     @GET
     @Produces(MediaType.TEXT_PLAIN)
-    public String hello() {
-        return "Hello World";
+    public String hello(@HeaderParam("Accept-Language") String lang) {
+        System.out.println("------------");
+        System.out.println(lang);
```

Comment:
USE A LOGGER :)

Reply:
And appropriate log levels. Always ask yourself: Do you want to sift through millions of lines of

```
------------
de
```

while searching for a real problem. So while some information might be helpful while debugging (hint hint), other might never be useful and other might need to be rephrased to be actually helpful.